### ISSUE TITLE
Is this product dead?

### ISSUE THREAD

**Issue by palefire19** [nylas/nylas-mail#3564]
It increasingly appears as if Nylas is basically giving up on Nylas Mail...after converting from N1 barely 2 months ago?

Anyone from Nylas, are you out there? Can you at least respond to one ticket on Github and give us a sense of whether you plan to continue developing and providing support for this product?

**Comment by DmitryLoki** [comment]
from the next issue in the list:

> Unfortunately, we are no longer providing help support for Nylas Mail or N1. As a company, Nylas is focused on our API products to help other developers build email, calendar, and sync capabilities into their own applications.

**Comment by Bergg** [comment]
That's a real shocker but it explains the absence of reactions to the pile of issues.

**Comment by mayurdw** [comment]
1000 people have forked it, I am sure someone must be updating it. We just need to find that someone who is trying to fix things

**Comment by anechol** [comment]
Oh man I _literally_ just installed this last night. Was looking for a good replacement for Thunderbird. What a shame. I really like this UI too, but it is ungodly slow retrieving my mail, and it's not like I have thousands of messages either.

**Comment by DmitryLoki** [comment]
@anechol the same shit ) replaced Thunderbird a week ago. I think I have to come to their office and check what's going on - there are pretty close to our :)

**Comment by mayurdw** [comment]
@DmitryLoki do it. Or at least make them transfer the contributor rights to someone who might wanna continue the project and commit other people's changes

**Comment by ronilaukkarinen** [comment]
Damn. Have been watching few of the issues without progress in half a year. Just downgraded just for this reason, would have liked to continue using Nylas Mail... bummer.

**Comment by sotayamashita** [comment]
Is this still maintained by Nylas?

**Comment by lxalln** [comment]
@sotayamashita in theory, but they have shown no sign nor intention to do anything with the OSS project. They have still been making releases.

**Comment by sotayamashita** [comment]
@lxalln I see. Thank you.

**Comment by mayurdw** [comment]
The CEO of Nylas replied to tweet with this
![image](https://cloud.githubusercontent.com/assets/21151744/26426485/e38fe474-412c-11e7-998a-e07b438119cc.png)

**Comment by danielmelogpi** [comment]
I'd be happy and try to help maintain this tool, as it has helped a lot in organizing my stuff.
Should we create a new organization and for the project there? May be the issues (and the precious pull requests) can be transfered to there as well?

**Comment by mayurdw** [comment]
@danielmelogpi yeah that would be good. I want to get involved with maintaining this as well but when I saw that no one is maintaining I figured what can I do on my own

**Comment by sotayamashita** [comment]
@danielmelogpi I can help you for code and translation :computer: :world_map:

**Comment by mayurdw** [comment]
great @danielmelogpi and @sotayamashita are in, I wanna do this as well so we have three people keen on picking this up

**Comment by dweremeichik** [comment]
@danielmelogpi I don't think that the PRs can be cloned, but the issues can. See this thread: https://github.com/holman/ama/issues/413. There are two tools there; the python tool can 'clone' the PRs but in the destination repo they are turned into issues.

**Comment by danielmelogpi** [comment]
I gave this a shot and forked the project to [https://github.com/nylas-mail-lives/nylas-mail](https://github.com/nylas-mail-lives/nylas-mail). 

I still need to study it in my spare time and I imagine you guys are in the same boat.
@mayurdw, @sotayamashita and @dweremeichik I sent you invites for membership. 
May be we can try to set this in motion!

**Comment by mayurdw** [comment]
Yay!! @danielmelogpi thanks. I have just joined it

**Comment by danielmelogpi** [comment]
Let's try slack then
https://join.slack.com/nylasisalive/shared_invite/MTg4MzQzMzY3NDEyLTE0OTU3NTQwNDMtYzhhZmE3NDM0NA

**Comment by sotayamashita** [comment]
@danielmelogp I appreciate you creating org. I have joined. :smile:

**Comment by danielmelogpi** [comment]
Anyone else willing to help us organize this endevour, please, use the link above and join us. We still need to understand the next steps and also, to understand this codebase to start working :)

**Comment by danielmelogpi** [comment]
@jstejada , would you be able to help us understand a good strategy to contribute? You sure have a lot of commits there and may be you can help us out

**Comment by palefire19** [comment]
Hi guys - I would love to help out here, but I'm not a developer. I do think I could serve as a reasonably useful product manager / ticket prioritizer, if that would be a helpful contribution...

**Comment by danielmelogpi** [comment]
@palefire19 , join us at slack :)

**Comment by sachinag** [comment]
Heya!  Congrats on the fork.  Mark sure that you're able to bring in any updates we make the to repo as you start your work.

**Comment by emorikawa** [comment]
Hello everyone! I work at Nylas and am extremely sorry for the lack of communication leading to this issue! Our engineering team has been heavily tasked on our enterprise API product for the past couple of weeks but we have not forgotten about Nylas Mail!

We'd like to bring the community in closer and do everything we can to establish and support community maintainers. We’re internally working through the proper governance issues right now, but to begin, I’d love to start chatting with at least @mayurdw and @danielmelogpi plus whoever else is interested. I’ll be posting more updates about this over the next several days.

**Comment by emorikawa** [comment]
Hello everyone! I work at Nylas and am extremely sorry for the lack of communication leading to this issue! Our engineering team has been heavily tasked on our enterprise API product for the past couple of weeks but we have not forgotten about Nylas Mail!

We'd like to bring the community in closer and do everything we can to establish and support community maintainers. We’re internally working through the proper governance issues right now, but to begin, I’d love to start chatting with at least @mayurdw and @danielmelogpi plus whoever else is interested.

I’ll be posting more updates about this over the next several days. In the meantime, either post here or reach out to me at evan@nylas.com if you’re interested in being involved in this process.

**Comment by alberto2000** [comment]
Hey guys I would like to help out too here. I'm experienced in react.js, node, electron and am also quite handy in responsive design, scss, compass/bourbon and all the faff. I think this is a great product with great potential and we should take this chance and make email great again. pun intended.

**Comment by palefire19** [comment]
@alberto2000 join us on Slack https://join.slack.com/nylasisalive/shared_invite/MTg4MzQzMzY3NDEyLTE0OTU3NTQwNDMtYzhhZmE3NDM0NA

**Comment by MickaelBergem** [comment]
@palefire19 I'm interested to join the Slack too, even if I'm not experienced with Node/Electron.
Looks like the Slack is invite-only for now so I can't join :/

**Comment by danielmelogpi** [comment]
@MickaelBergem and others, I just found out that the invitation link expires after 30 days. Will try to reach out slack team to get it reenabled.

**Comment by danielmelogpi** [comment]
Just received a new link from Nylas:
https://join.slack.com/nylasisalive/shared_invite/MTkyNTkyNzMwODA1LTE0OTY2MjIyMDQtMWNhNzI0NWM4Mg

Also, updating my first message to avoid mistakes

**Comment by iansinnott** [comment]
Hey everyone, is there still discussion going on in that Slack group? I tried the link but it's expired

**Comment by dweremeichik** [comment]
Indeed there is. I will ping @danielmelogpi to see if we can get that link sorted out.

**Comment by dweremeichik** [comment]
@iansinnot I sent a slack invite to the email listed on your profile.

**Comment by iansinnott** [comment]
Thanks @dweremeichik :)

**Comment by rgaiacs** [comment]
Can I have a invitation to Slack?

**Comment by dweremeichik** [comment]
@rgaiacs done.

**Comment by mitchazj** [comment]
Would also love a Slack invite :)

**Comment by dweremeichik** [comment]
@mitchazj give this link a shot.
https://join.slack.com/nylasisalive/shared_invite/MjAzMDE1NTU0MDM2LTE0OTgyNzM3MTItODY4OWNlNTdkMw

**Comment by adriantombu** [comment]
So if I understand it right, paid user threw money out the window with a software that is not updated anymore. Love it.

**Comment by dweremeichik** [comment]
@adriantombu incorrect, Nylas is still working on the product internally and releasing "updates". They are currently not including feedback and pull requests from this community however. Also, the fork that was made from this project still relies on the Nylas API (Nylas pays for those servers to run).

**Comment by adriantombu** [comment]
Well I don't remember the last time I had the privilege of getting an update, that must have been like 2 months ago...

**Comment by danielbayley** [comment]
Fairly annoying situation! I was an early adopter, from before we had to sign up for _another_ account and everything went all ‘cloud’.

@danielmelogpi @mayurdw @sotayamashita @dweremeichik So is _[nylas-mail-lives]_ now _the_ main fork as far as this community is concerned?

I would like to see the project not die, and willing to contribute when I can… I work on/in Atom all the time so I know my way around Electron and any remnants of the CoffeeScript (which I love…) codebase, as this app was a fork of Atom early on. Incidentally, [CoffeeScript 2] (which compiles to ES6) is pretty much ready now in it’s 3rd beta.

Anyway, some things I would like to see happen:

* Decouple the app from requiring _Nylas_ altogether (especially removing the need to sign into a separate account before signing in again to your actual email account!)
* Refactor ‘send later’/‘snooze’ etc to work independently/locally (these could use a launch agent on macOS for example)
* Reinstate and expand mail rules for automatic sorting of mail
* Fix the Markdown composer! (PR here thanks to @Morsby  https://github.com/nylas/N1-Markdown-Composer/pull/27)
* Rename the new forked project to something else, maybe ‘Open Mail’ or something
* It’s own GitHub project/org page
* Proper package management with a cli, probably based on npm

Is it worth joining the new #Slack group? Wouldn’t using the [existing Nylas one] have been a better option? Or Gitter even…

[nylas-mail-lives]: https://github.com/nylas-mail-lives/nylas-mail
[CoffeeScript 2]: http://coffeescript.org/v2
[existing Nylas one]: http://nylas-slack-invite.herokuapp.com

**Comment by dweremeichik** [comment]
We have seen a good bit of interest from the community on a number of your bullet points. Unfortunately for a majority of them, no one has stepped up to tackle them as of yet. 

It is definitely worth it to join the new slack channel. The new one was created due to the circumstances surrounding this repository (not knowing what was/is going to happen).

I would recommend creating issues in the fork for feature requests that you would like to see worked on. Note that the fork relies heavily on the community, so if you see something you would like to work on, mention it in the issue.

Contributors of any caliber are welcome!

**Comment by step21** [comment]
@danielbayley at least the first point should be easier since they already moved the main e-mail part away from the cloud.

**Comment by amccloud** [comment]
@dweremeichik  @emorikawa @GM-Polyakov @jackiehluo @jstejada @khamidou @necrobuffalo @pfista @spang

I've been trying to contact Nylas for a week now. As an early adopter and a paying customer, I'm very upset with how you're communicating this change. My Nylas email account has been broken for a week and I guess your response is 

![image](https://user-images.githubusercontent.com/38463/28539394-55478b2c-7066-11e7-850d-074755f73c51.png)

You can imagine that as a developer I'm now less likely to consider your new venture into developer tools if this is how you communicate.

**Comment by reyman** [comment]
Same here @amccloud , i'm really really very upset.

**Comment by dweremeichik** [comment]
@amccloud I don't work for Nylas, I do contribute to the new fork though.

https://github.com/nylas-mail-lives/nylas-mail/

What issue are you seeing with your client? The fork may have fixed it already, if it isn't, you are more than welcome to try and tackle the issue in the fork. Or at least create the issue there.
Mind linking the issue in?

**Comment by danielmelogpi** [comment]
@amccloud , I recommend you keep trying contact with Nylas Support team, as none of us in this thread, as far as I read it, works with them. 
If it comes to it, I suggest you cancel your subscription with them. It is, indeed, very upsetting to pay and not receive your goods anymore.

**Comment by bengotow** [comment]
Hey folks - I'm one of the original authors of Nylas Mail and worked at Nylas until last winter. As you've all mentioned, Nylas has pulled engineers off of the mail client and shifted focus to the API products that are the primary revenue drivers for the company. As frustrating as this move has been (for me as well), it was definitely necessary. Nylas is a venture-backed startup and startups need to make money.

I've always loved Nylas Mail, and I'm working on a fork that will fix longstanding bugs and put it on a better foundation for continued development. I'm actually re-writing the entire mail sync engine in C++ for better performance and battery life, and deploying backend infrastructure to run snooze, send later, etc.

You can download the new version (called Mailspring) here: https://getmailspring.com/, or find it on GitHub at https://github.com/Foundry376/Mailspring/.

**Comment by laughedelic** [comment]
It would be wonderful if the two forks had each other in mind to come to some consensus later.

**Comment by mjchamplin** [comment]
Somehow I'd missed all this discussion until this morning. I've found Nylas to be frustrating since the switch from N1 earlier this year. The product simply isn't working to the level needed to be considered a viable email client. I'd be fine with this, and just move on, except they're also now trying to get me to pay for features that used to be free (really, I've reached my maximum number of email snoozes for the week?)

Not meaning to rant, and I respect that a company has to make money, but this was all very poorly handled. The complete lack of transparency is really, really frustrating.
